packages/contentstack-import/src/lib/import/assets.js

```diff
@@ -60,9 +60,9 @@ importAssets.prototype = {
     mkdirp.sync(mapperDirPath);
 
     return new Promise(function (resolve, reject) {
-      if (self.assets === undefined) {
+      if (self.assets === undefined || Object.keys(self.assets).length === 0) {
```

Comment:
@abhinav-from-contentstack Can you reverify this condition also?

it checks that **if (some_variable === undefined || Object.keys(some_variable).length === 0)** right. Ultimately if the first condition fails it's going to jump into or condition **(Object.keys(some_variable).length === 0)** to validate right, in that case it's going to fail. Because **Object.keys(undefined).length** is not a valid code.

What if my **some_variable(asset/any)** is ideally undefined, All we know is variables can be undefined as well. So better handle it to prevent such an issue.